src/lammpsio/snapshot.py

```diff
@@ -437,6 +439,33 @@ def has_mass(self):
         """
         return self._mass is not None
 
+    @property
+    def bonds(self):
+        """:class:Bonds: bonds."""
+        if not self.has_bonds():
+            pass
```

Comment:
This code doesn't do anything, and I think it's OK for `bonds` to return as `None`.
```suggestion
```
The alternative would be to throw an error if `bonds` is accessed but `not self.has_bonds()`. I am not sure if that is desirable behavior, though.